src/storage/heap_file.c

```diff
@@ -16614,42 +16612,22 @@ heap_attrinfo_set_uninitialized_global (THREAD_ENTRY * thread_p, OID * inst_oid,
 }
 
 /*
- * heap_get_hfid_from_class_oid () - get HFID from class oid
- *   return: error_code
- *   class_oid(in): class oid
- *   hfid(out):  the resulting hfid
- */
-int
-heap_get_hfid_from_class_oid (THREAD_ENTRY * thread_p, const OID * class_oid, HFID * hfid)
-{
-  int error_code = NO_ERROR;
-
-  error_code = heap_hfid_cache_get (thread_p, class_oid, hfid, NULL);
-  if (error_code != NO_ERROR)
-    {
-      ASSERT_ERROR ();
-      return error_code;
-    }
-
-  return error_code;
-}
-
-/*
- * heap_get_hfid_and_file_type_from_class_oid () - get HFID and file type for class.
+ * heap_get_class_info () - get HFID and file type for class.
  *
- * return          : error code
- * thread_p (in)   : thread entry
- * class_oid (in)  : class OID
- * hfid_out (out)  : output heap file identifier
- * ftype_out (out) : output heap file type
+ * return             : error code
+ * thread_p (in)      : thread entry
+ * class_oid (in)     : class OID
+ * hfid_out (out)     : output heap file identifier
+ * ftype_out (out)    : output heap file type
+ * classname_out (out): output classname
  */
 int
-heap_get_hfid_and_file_type_from_class_oid (THREAD_ENTRY * thread_p, const OID * class_oid, HFID * hfid_out,
-					    FILE_TYPE * ftype_out)
+heap_get_class_info (THREAD_ENTRY * thread_p, const OID * class_oid, HFID * hfid_out,
+		     FILE_TYPE * ftype_out, char *classname_out)
```

Comment:
what happens with classname_out value after heap_get_class_info scope is finished?